### DISCUSSION TITLE
любые ваши идеи по игре

### DISCUSSION THREAD

**Discussion by Pigzinzspace** [godot-engine-ru/shooter-3d#18] [Ideas]
**содержание документа**  Сюда в обсуждение, любой желающий может писать любые свои идеи по игре, по любому ее аспекту. Наша структура данных все еще несовершенна и не охватывает все темы. Не создавайте новые темы, просто напишите идею сюда, мы ее обсудим, и если она пройдет обсуждение она переедет в нужные разделы.

перед тем как что то писать сюда, прочитайте эти темы чтоб понять что точно в игру не попадет и что там уже есть

важная дискуссия о структуре файлов проекта
https://github.com/godot-engine-ru/shooter-3d/discussions/15 
гейиплэй
https://github.com/godot-engine-ru/shooter-3d/discussions/12
контролер игрока
https://github.com/godot-engine-ru/shooter-3d/discussions/13
оружие и гаджеты
https://github.com/godot-engine-ru/shooter-3d/discussions/11
монстры
https://github.com/godot-engine-ru/shooter-3d/discussions/9
список миссий
https://github.com/godot-engine-ru/shooter-3d/discussions/7

**Comment by soundsbeard** [comment]
идея для названия:

Culled
(каллд) как на заводах и фабриках происходит отбраковка, так и главный герой был отбракован с "конвеера" вируса.

**Reply by Pigzinzspace** [reply]
поподробнее, а все я понял 
ну да очевидно что с ним что-то не чисто, либо у него врожденный  иммунитет, либо он как то заряжен вакциной, иначе как все заразились а он нет, даже после непосредственного контакта и ранений укусов.

единственное что Калд это не благозвучно (название кал игре гарантировано), может какие то другие варианты

**Reply by me2beats** [reply]
Соглы не благозвучно.
И потом нужно именно с английского названия начинать?
С русского думаю лучше

**Reply by Einshtein12** [reply]
Давайте создадим настройку rus/eng и перевод текста и озвучки(если таковые имеются)

**Reply by Pigzinzspace** [reply]
добавил пункт по локализацию в геймплэй
